### PR TITLE
Add documentation for TubeTrailMesh and RibbonTrailMesh

### PR DESCRIPTION
**Pull request by MewPurPur**
Brings class reference documentation up from 87% to 87.2% or something idk it's neverending.

### CODE REVIEW

**Comment by QbieShay** [comment]
Heroic effort :bow:

**Comment by adamscott** [comment]
That branch name. :rofl:

File: doc/classes/TubeTrailMesh.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TubeTrailMesh" inherits="PrimitiveMesh" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Represents a straight tube-shaped [PrimitiveMesh] with variable width.
 	</brief_description>
 	<description>
+		[TubeTrailMesh] represents a straight tube-shaped mesh with variable width. The tube is composed of a number of cylindrical sections, each with the same [member section_length] and number of [member section_rings]. A [member curve] is sampled along the total length of the tube, meaning that the curve determines the radius of the tube along its length.
 	</description>
 	<tutorials>
```
Comment (clayjohn) [inline doc/classes/TubeTrailMesh.xml:9]: Should link to https://docs.godotengine.org/en/latest/tutorials/3d/particles/trails.html and https://docs.godotengine.org/en/latest/tutorials/3d/particles/index.html

File: doc/classes/RibbonTrailMesh.xml [hunk 2]
```diff
@@ -1,29 +1,39 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RibbonTrailMesh" inherits="PrimitiveMesh" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Represents a straight ribbon-shaped [PrimitiveMesh] with variable width.
 	</brief_description>
 	<description>
+		[RibbonTrailMesh] represents a straight ribbon-shaped mesh with variable width. The ribbon is composed of a number of flat or cross-shaped sections, each with the same [member section_length] and number of [member section_segments]. A [member curve] is sampled along the total length of the ribbon, meaning that the curve determines the size of the ribbon along its length.
 	</description>
 	<tutorials>
```
Comment (clayjohn) [inline doc/classes/RibbonTrailMesh.xml:9]: Should link to https://docs.godotengine.org/en/latest/tutorials/3d/particles/trails.html and https://docs.godotengine.org/en/latest/tutorials/3d/particles/index.html

File: doc/classes/RibbonTrailMesh.xml [hunk 3]
```diff
@@ -1,29 +1,39 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RibbonTrailMesh" inherits="PrimitiveMesh" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Represents a straight ribbon-shaped [PrimitiveMesh] with variable width.
 	</brief_description>
 	<description>
+		[RibbonTrailMesh] represents a straight ribbon-shaped mesh with variable width. The ribbon is composed of a number of flat or cross-shaped sections, each with the same [member section_length] and number of [member section_segments]. A [member curve] is sampled along the total length of the ribbon, meaning that the curve determines the size of the ribbon along its length.
```
Comment (clayjohn) [inline doc/classes/RibbonTrailMesh.xml:7]: This should mention that RibbonTrailMesh is intended to be used with the particles trail feature

File: doc/classes/TubeTrailMesh.xml [hunk 4]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TubeTrailMesh" inherits="PrimitiveMesh" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Represents a straight tube-shaped [PrimitiveMesh] with variable width.
 	</brief_description>
 	<description>
+		[TubeTrailMesh] represents a straight tube-shaped mesh with variable width. The tube is composed of a number of cylindrical sections, each with the same [member section_length] and number of [member section_rings]. A [member curve] is sampled along the total length of the tube, meaning that the curve determines the radius of the tube along its length.
```
Comment (clayjohn) [inline doc/classes/TubeTrailMesh.xml:7]: Same as with the RibbonTrailMesh, particle trails should be mentioned here

**Review by clayjohn — APPROVED** [review]
Looks good to me! thank you

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Behold: a merge! Thanks for completing the docs :)

**Pull request merged**